packages/angular/build/src/tools/esbuild/angular/compiler-plugin.ts

```diff
@@ -62,7 +63,17 @@ export function createCompilerPlugin(
       const preserveSymlinks = build.initialOptions.preserveSymlinks;
 
       // Initialize a worker pool for JavaScript transformations
-      const javascriptTransformer = new JavaScriptTransformer(pluginOptions, maxWorkers);
+      let cacheStore;
+      if (pluginOptions.sourceFileCache?.persistentCachePath) {
+        cacheStore = new LmbdCacheStore(
+          pluginOptions.sourceFileCache.persistentCachePath + '/angular-compiler.db',
```

Comment:
I guess to make the 5 windows users happy, this could be:
```ts
path.join(pluginOptions.sourceFileCache.persistentCachePath, 'angular-compiler.db')
```

Reply:
Windows actually supports both path separators; but, yes it could.